### ISSUE TITLE
Rate Now

### ISSUE THREAD

**Issue by BMwanza** [nraboy/nativescript-ratings#5]
Just a question with this plug in.

I would like to add it to my app but I do not want to come across as spamming users. My question is, if a user were to hit Rate Now, will that stop the prompt from coming up once it reaches the specified number in the counter?

**Comment by shiv19** [comment]
Yes, it wouldn't show again if the user doesn't want to rate. But if user chooses remind later, then it will show again after x number of launches
